src/Organisms/DogData/DogData.js

```diff
@@ -29,18 +50,6 @@ const DogData = ({ id }) => {
     // danymi właściciela
   };
 
-  const dogData = {};
-  Object.keys(DOG_DATA_TEMPLATE).forEach(
-    (prop) => (dogData[DOG_DATA_TEMPLATE[prop]] = dog[prop] || 'brak danych'),
-  );
-
-  const { dateOfBirth, sex } = DOG_DATA_TEMPLATE;
-
-  //Date of birth convert to string
-  dogData[dateOfBirth] = `${getDataFormatDdMonthYyy(dogData[dateOfBirth])}`;
-  //Sex convert to string
-  dogData[sex] = doggoSex[dogData[sex]];
-
   return (
     <ColumnWrapper paddingLeftRight={1}>
       <SpecialButtonsContainerStyled>
```

Comment:
we have participant-data page on development, we can make a navigation from dog-data, to praticipant-data

Reply:
and maybe it will be better to change /participant-data to /participant-data/:userId